Formula/hyper.rb

```diff
@@ -2,8 +2,8 @@ class Hyper < Formula
   desc "Client for the Hyper_ cloud service"
   homepage "https://hyper.sh"
   url "https://github.com/hyperhq/hypercli.git",
-    :tag => "v1.10",
-    :revision => "603e437099a0dc94943ee10219b35d57221262fa"
+    :tag => "v1.10-1",
```

Comment:
Can you add an explicit version here? `brew` has trouble sometimes with `<number>.<number>-<number>` version formats.

Underneath the `:revision` line below add:
```ruby
version "1.10-1"
```
Thanks.